internal/config/loader.go

```diff
@@ -146,7 +146,7 @@ func LoadConfig(useRegistry string, profile string, confDir string) (configurati
 		}
 	}
 
-	fmt.Println(registryMsg)
+	fmt.Fprintln(os.Stdout, registryMsg)
```

Comment:
it would be nice if you could put this change into a different commit

Reply:
I use different commit now